### DISCUSSION TITLE
eslint-config-epicstack eslint-plugin-epicstack

### DISCUSSION THREAD

**Discussion by onemen** [epicweb-dev/epic-stack#158] [Ideas]
@kentcdodds 

we can use `eslint-config-epicstack`

let's make a wish list for epic-stack opinionated lint rules: typescript, prettier, etc.

we can impalement any non-exist rule in `eslint-plugin-epicstack`

it is also possible to integrate `eslint-config-prettier` and `@remix-run/eslint-config`

maybe even `@epic-stack/config` ?

**question**: 
what do you think about moving most if not all config files to a config folder.
I know it can be done for eslint, prettier maybe other...

**Comment by ananni13** [comment]
The [current eslint config](https://github.com/epicweb-dev/epic-stack/blob/main/.eslintrc.cjs) is already pretty minimal and only adds very few things on top of `@remix-run/eslint-config`. I think creating a custom package for this will add unnecessary complexity for not really much benefit.

Also, after you start adding custom packages you begin to blur the line between a template/starter and just another framework to learn and maintain. It comes down to what direction this project wants to go.

Just my two cents 🙂

**Comment by kentcdodds** [comment]
I created this org: https://www.npmjs.com/org/epic-web

There are a number of things in the Epic Stack that could go into an OSS package under there. I'm not really keen on adding a bunch of ESLint rules at the moment. I'm pretty happy with the light-touch of the linting config currently. But I'll keep this in mind.
